apps/api/src/common/trim.pipe.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Injectable, PipeTransform, ArgumentMetadata } from '@nestjs/common';
+
+@Injectable()
+export class TrimPipe implements PipeTransform {
```

Comment:
Does this mean FE doesn't need to trim values?

Reply:
This is a very nestjs/coupled way of trimming. I don't think it's worth the effort to make it generic

Reply:
Sorry that was a weird question, does this just trim incoming fields?

Reply:
yeah, its a nice to have. We can remove it if there's no benefit.